keras/utils/test_utils.py

```diff
@@ -116,6 +116,9 @@ def layer_test(layer_cls, kwargs={}, input_shape=None, input_dtype=None,
 
     # test as first layer in Sequential API
     layer_config = layer.get_config()
+    # deals with data_format in flatten
+    if 'data_format' in kwargs:
+        layer_config['data_format'] = kwargs['data_format']
```

Comment:
Revert this change and add support for `data_format` in `get_config` for this layer

Reply:
Forgot about that func, I've updated to reflect your comments.